src/enigma2/TimeshiftBuffer.cpp

```diff
@@ -39,6 +39,10 @@ TimeshiftBuffer::~TimeshiftBuffer(void)
   }
   if (m_filebufferReadHandle)
     XBMC->CloseFile(m_filebufferReadHandle);
+  
+  if (!XBMC->DeleteFile(m_bufferPath.c_str()))
```

Comment:
This can be troublesome, I did this in pvr.vbox:  https://github.com/kodi-pvr/pvr.vbox/blob/master/src/timeshift/FilesystemBuffer.cpp (see the undef, can't link from mobile)

Reply:
Thanks, I’ll add the undef for Windows. @ksooo, looks like I’ll have to do something platform dependent now anyways unless you have a better suggestion?

Reply:
no better suggestion atm :-/

Reply:
As this currently compiles, you don't have to care about it.
It's needed in pvr.vbox as windows headers are pulled in somewhere.

Reply:
@Jalle19 so was the error you had at runtime or compile time?

Reply:
I honestly can't remember